services-js/registry-certs/pages/_document.tsx

```diff
@@ -91,7 +78,7 @@ export default class extends Document {
                     client: {
                       javascript: {
                         source_map_enabled: true,
-                        code_version: "${rollbarVersion || buildId}",
+                        code_version: "${rollbarVersion}",
```

Comment:
Keep the `buildId`

Reply:
The value was coming from `__NEXT_DATA__` - what should I assign it to instead?

Reply:
Keep using `__NEXT_DATA__`. It will still be on the `this.props` that `Document` has.